public/scripts/hackatime-install.sh

```diff
@@ -36,78 +50,24 @@ configure_wakatime() {
     echo
 }
 
-install_code_command() {
```

Comment:
didn't this automatically install code to bin?

Reply:
In my experience no, after running the script there was no new file containing the paths(was a new mac, so nth had been setup yet thus there we're no existing paths so creating the code command should have resulted in a new file) and on my other mac which alr had other paths stored, when I went to the zhsprofile file(and its equivalent files that may store paths) I couldn't locate the code command, had to add it manually. I did verify on both laptops that my script works. Maybe you want to integrate it as a backup? Also note that I didn't get any error stating that the code command couldn't be added to path.